### PR TITLE
Fixes #29738 - Extract strings from plugin webpack dir

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

**Comment by adamruzicka** [comment]
Was there a reason for different glob patterns for core and plugins?

File: lib/tasks/gettext.rake [hunk 1]
```diff
@@ -30,7 +32,7 @@ begin
       end
 
       def files_to_translate
-        Dir.glob("#{@engine.root}/{app,db,lib,config,locale}/**/*.{rb,erb,haml,slim,rhtml,js}")
+        Dir.glob(File.join(@engine.root, FILE_GLOB))
```
Comment (ekohl) [inline lib/tasks/gettext.rake]: Doesn't this also work (since `root` is a Pathname):
```suggestion
        @engine.root.glob(FILE_GLOB)
```

Comment (adamruzicka) [reply lib/tasks/gettext.rake]: It does, thanks

**Review by tbrisker — APPROVED** [review]
Thanks @adamruzicka !

**Pull request merged**